src/components/BorrowerProfile/UpdateDetails.vue

```diff
@@ -30,7 +30,7 @@
 						:share-message="shareMessage"
 						:share-url="$route.path + '#updates'"
 						variant="link"
-						utm-campaign="bp_update_share"
```

Comment:
All our other utm-campaigns are "social_share_<something>" so just modifying this one